react-components/src/utilities/FdmSDK.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type CogniteClient } from '@cognite/sdk';
+
+type InstanceType = 'node' | 'edge';
+
+export type Item = {
+  instanceType: InstanceType;
+} & DmsUniqueIdentifier;
+
+export type Source = {
+  type: 'view';
+  version: string;
+} & DmsUniqueIdentifier;
+
+export type DmsUniqueIdentifier = {
```

Comment:
`DmsUniqueIdentifier` not used outside the file

Reply:
The same as above.